cypress/integration/tests/upgrade/preparation.ts

```diff
@@ -0,0 +1,149 @@
+/*
+Copyright © 2021 the Konveyor Contributors (https://konveyor.io/)
```

Comment:
Looks good , this file can be renamed to create_test_data.ts

Reply:
Fixed